src/containers/org/OrgDataSetContainer.js

```diff
@@ -24,13 +32,29 @@ import {
   SpaceBetweenGrid,
   StackGrid,
 } from '../../components';
+import { getOrgDataSetSize } from '../../core/edm/actions';
 import { FQNS } from '../../core/edm/constants';
-import { selectDataSetSchema, selectOrgDataSet, selectOrganization } from '../../core/redux/selectors';
+import {
+  selectDataSetSchema,
+  selectOrgDataSet,
+  selectOrgDataSetColumns,
+  selectOrgDataSetSize,
+  selectOrganization
+} from '../../core/redux/selectors';
 import { Routes } from '../../core/router';
 import { isAtlasDataSet } from '../../utils';
 
+const { BLUE } = Colors;
+
 const { getPropertyValue } = DataUtils;
 const { isNonEmptyString } = LangUtils;
+const { isValidUUID } = ValidationUtils;
+
+const CountBadge = styled(Badge)`
+  background: ${BLUE.B300};
```

Comment:
The color doesn't match Figma, but Figma isn't using the standard colors 🤔

Reply:
I'll put a note in figma